src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -37,414 +36,413 @@
+                .filter(port -> Objects.equals(port.getName(), "http-api"))
+                .findFirst()
+                .orElseThrow(() -> new ResourceNotFoundException("service port not found"))
+                .getPort();
+        LOG.debug("computed service port is {}, clusterIp is {}", svcPort, clusterIP);
+
+        String sideCarSvcUrl = "tcp://".concat(clusterIP).concat(":").concat(String.valueOf(svcPort));
+        LOG.debug("computed sidecar service url is {}", sideCarSvcUrl);
+
+        return sideCarSvcUrl;
+
     }
-    if (pods.isEmpty()) {
-      LOG.info("no pod found with the following details {}", labels);
-      return isAvailable;
+
+    Map<String, String> searchCriteriaLabel(Environment environment) {
+        return Map.of(MANAGED_BY, SPASHIP,
+                WEBSITE, environment.getWebsiteName().toLowerCase(),
+                ENVIRONMENT, environment.getName().toLowerCase());
+    }
+
+    private OperationResponse applyDeleteResourceList(Environment environment, KubernetesList resourceList) {
+        LOG.debug("applying delete on resources");
+        // Should we delete the pvs as well? will that be a good idea?
+        var isDeleted = ocClient.resourceList(resourceList).inNamespace(environment.getNameSpace()).delete();
+        environment.setOperationPerformed(true);
+        var or = OperationResponse.builder().environment(environment)
+                .sideCarServiceUrl("NA")
+                .originatedFrom(this.getClass().toString());
+        or.status(3);
+        if (isDeleted.isEmpty())
+            or.status(0).errorMessage("unable to delete the resources");
+        return or.build();
+    }
+
+    private KubernetesList buildK8sResourceList(Environment environment) {
+        LOG.info("inide buildK8sResourceList the environment value is {}", environment);
+        Map<String, String> templateParameters = Map.of(
+                "WEBSITE", environment.getWebsiteName().toLowerCase(),
+                // "TRACE_ID", environment.getTraceID().toString().toLowerCase(),
+                "ENV", environment.getName().toLowerCase(),
+                "WEBSITE_VERSION", environment.getWebsiteVersion().toLowerCase(),
+                "DOMAIN", domain,
+                "APP_INSTANCE_PREFIX", appInstance,
+                "STORAGE_CLASS", ConfigProvider.getConfig().getValue("storage.class", String.class),
+                "NS", environment.getNameSpace(),
+                "SHARD", routerShard,
+                "ROUTER_DOMAIN", routerDomain,
+                "CMDB_CODE", environment.getCmdbCode()
+        );
+        LOG.debug("building KubernetesList, templateParameters are as follows {}", templateParameters);
+
+        KubernetesList kubernetesList = null;
+        try {
+            kubernetesList = ocClient
+                    .templates()
+                    .inNamespace(environment.getNameSpace())
+                    .load(Operator.class.getResourceAsStream("/openshift/environment-template.yaml"))
```

Comment:
could you please add this path to a variable? 

private final String OPENSHIFT_ENVIRONMENT_PATH = "/openshift/environment-template.yaml";